doc/security/specs/device_life_cycle/_index.md

```diff
@@ -0,0 +1,646 @@
+
+An additional set of life cycle states are used to encode the device ownership
+state. A device that has been personalized with a unique Silicon Creator
+Identity, can be provisioned with a Silicon Owner Identity and Credentials. This
+enables the Silicon Owner to run signed code on the device. The process of
+assigning a Silicon Owner to a device is referred to as Ownership Transfer.
+Depending on the product SKU configuration, a device may support a single and
+permanent ownership transfer or multiple ones. The multiple ownership transfer
+use case has additional state control requirements which are described in more
+detail in later sections.
+
+## Architecture
+
+Life cycle states are defined by a combination of OTP and flash. OTP state
+transitions cannot be reverted, while flash defined states are meant to be
+'flexible', especially when ownership transfer is required. This implies that
+when OTP states are transitioned from A to B, we cannot go back to A from B.
+However, it is still possible to transition from B to C, assuming such a
+transition is legal and allowed by the life cycle definition.
+
+See the table below for more details.
+
+<img src="device_life_cycle_fig1.svg" alt="Fig1" style="width: 800px;"/>
+
+In the diagram above,
+
+*   **BLACK** arcs represent unconditional OTP transitions. Specifically, this
+    means that the state will transition as long as it was requested; no
+    additional authorization is necessary.
+*   **ORANGE** arcs represent conditional flash transitions. Specifically, the
+    transition requires either specific value patterns or device specific values
+    to be programmed. Since flash states are managed by software, this is not
+    further covered in this specification.
+*   **RED** arcs represent conditional transitions. These transitions require
+    the user to supply specific device-unique values and meet specific
+    conditions. These transitions cannot be reversed.
+
+Device life cycle is split into two domains.
+
+*   Manufacturing state
+*   Identity & Ownership state
+
+Manufacturing state largely refers to the presence of debug functions and
+liveness of the device. This is closely connected to manufacturing testing
+status as well as desired debug visibility during development and production.
+
+Identity & Ownership state refers to the state of provisioned identities as
+defined by the provisioning flow.
+
+The life cycle management system splits the states into two because there are
+attributes defined by each that are not strictly related most of the time. By
+decoupling a single definition into two, more flexibility is allowed in the
+system.
+
+In addition, while certain states are defined by OTP (thus hard), other states
+are defined by flash. This gives the latter more flexibility in changing values
+to support the provisioning flow ownership transfer model.
+
+For example, DEV / PROD / RMA manufacturing states are defined in OTP states and
+cannot be exited once entered.
```

Comment:
Is this line accurate? The way I read it implies that neither DEV nor PROD can be exited once entered. But the diagram shows exist transitions into the RMA state. Then more pedantically, even the RMA state has an exit condition to both the SCRAP and INVALID states. 

It looks like the overall point you are trying to make is less about the states, but more about the directionality of the transitions. You can go between OWNED / UNOWNED repeatedly, but you can't go from RMA back to PROD, or PROD back to TEST. 

"For example, transitions into the DEV / PROD / RMA manufacturing states affect OTP states and therefore cannot be reversed once completed."

Reply:
SGMT. Updated wording. Done.